lib/miq_automation_engine/engine/miq_ae_engine/miq_ae_uri.rb

```diff
@@ -18,7 +18,7 @@ def self.query2hash(query)
       hash = {}
       query&.split('&')&.each do |a|
           k, v = a.split('=')
-          hash[CGI.unescape(k)] = CGI.unescape(v.to_s)
+          hash[URI.unescape(k)] = URI.unescape(v.to_s) # rubocop:disable Lint/UriEscapeUnescape
```

Comment:
Thanks so much @astrozzc for your work here. :-)

The (https://github.com/ManageIQ/manageiq-automation_engine/pull/412) PR from @gmcculloug addresses the URI rubocop errors here. 

You should be able to remove this file from your PR.  :-)

Reply:
My pleasure :) Got it, fixed 👍